recipe/build.sh

```diff
@@ -324,20 +324,20 @@ Mingw_w64_makefiles() {
     else
       mkdir miktex || true
       pushd miktex
-      MIKTEX_VER=2.9.7100
       # Fetch e.g.:
       # http://ctan.mines-albi.fr/systems/win32/miktex/tm/packages/url.tar.lzma
       # http://ctan.mines-albi.fr/systems/win32/miktex/tm/packages/mptopdf.tar.lzma
       # http://ctan.mines-albi.fr/systems/win32/miktex/tm/packages/inconsolata.tar.lzma
-        curl --insecure -C - -o ${DLCACHE}/basic-miktex-${MIKTEX_VER}.exe -SL https://miktex.org/download/ctan/systems/win32/miktex/setup/windows-x86/basic-miktex-${MIKTEX_VER}.exe || true
-        echo "Extracting basic-miktex-${MIKTEX_VER}.exe, this will take some time ..."
-        ${DLCACHE}/basic-miktex-${MIKTEX_VER}.exe --portable=${PWD} --unattended --no-registry &
-        local miktex_setup_pid=$!
-        while kill -0 "${miktex_setup_pid}" >/dev/null 2>&1 ; do
-          du -sc ./*
-          echo ========================
-          sleep 5
-        done
+        # FIXME: Newer MiKTeX installer does not finish on AppVeyor and Azure, somehow.
+        #   MIKTEX_VER=2.9.7100
+        #   curl --insecure -C - -o ${DLCACHE}/basic-miktex-${MIKTEX_VER}.exe -SL https://miktex.org/download/ctan/systems/win32/miktex/setup/windows-x86/basic-miktex-${MIKTEX_VER}.exe || true
+        #   echo "Extracting basic-miktex-${MIKTEX_VER}.exe, this will take some time ..."
+        #   ${DLCACHE}/basic-miktex-${MIKTEX_VER}.exe --portable=${PWD} --unattended --no-registry || exit 1
```

Comment:
This works on a local Windows machine. The progress is displayed with a GUI, though, so tough luck inspecting it on the CI...
The installer does something on the CI, as seen by https://github.com/conda-forge/r-base-feedstock/pull/84/commits/6e294637e322d21d59fe5acc20c60ee2022c9a60 which outputs:
```
Extracting basic-miktex-2.9.7100.exe, this will take some time ...
du: cannot access './*': No such file or directory
0	total
========================
11885	./texmfs
11885	total
========================
66684	./texmfs
66684	total
========================
147410	./texmfs
147410	total
========================
244796	./texmfs
244796	total
========================
373200	./texmfs
373200	total
========================
426704	./texmfs
426704	total
========================
437725	./texmfs
437725	total
========================
437725	./texmfs
437725	total
========================
...
```
but it obviously hangs somewhere...